lib/LANraragi/Utils/Plugins.pm

```diff
@@ -164,13 +183,14 @@ sub use_plugin {
         %pluginfo = $plugin->plugin_info();
 
         # Get the plugin settings in Redis
-        my @settings = get_plugin_parameters($plugname);
+        my %settings = get_plugin_parameters($plugname);
+        $settings{oneshot} = $input;
```

Comment:
To reply to the question in the PR description:  

I wouldn't merge `lrr_info` and `params`, I think it's useful to keep a clear distinction between data that's dynamically provided by the server depending on the archive in play (name, thumbnail/id/tags, etc), and static-ish plugin configuration set by the user in the settings.  

The oneshot parameter in more of a one-time parameter set by the user for the current archive, so it can fit in both hashes -- I think it's fine having it in `params` as a result.  

(although since we're keeping backwards compatibility here, they'll stay duplicated for a while? I wouldn't really mind having it stay in lrr_info only either.)